### ISSUE TITLE
CellNav Doesn't Acknowledge Column Filtering

### ISSUE THREAD

**Issue by CindyConway** [angular-ui/ui-grid#1654]
I have a grid where the keyboard navigation works fine. I filter a column, then the navigation to the next row using the up or down arrow doesn't work. The left and right arrow navigation still works as expected. This looks like it is related to issue #1410 (in that the index is used for row, and that does not reflect visible rows after the filter is applied.)
